dashboard/config/locales/scripts.en.yml

```diff
@@ -398,6 +398,8 @@ en:
               name: Dance Lab2
             lesson-53:
               name: Python Lab
+            lesson-54:
+              name: Web Lab 2
         algebraPD2b:
```

Comment:
super nit: Do we care that `Dance Lab2` doesn't have a space between "Lab" and "2" and `Web Lab 2` does? The name of the allthethings level doesn't matter, right?

Reply:
oh good point. These names are changeable, everywhere it matters it's all one word. "Web Lab 2" makes more sense to me because 2 is modifying "Web Lab", but I am open to changing it to make it consistent.

Reply:
As-is works for me!